demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -900,10 +896,11 @@ def filter_changed_files(self, files_string, tag='master', print_ignored_files=F
                 elif file_status.lower() == 'm':
                     changed_meta_files.add(file_path)
 
-            elif print_ignored_files and not checked_type(file_path, IGNORED_TYPES_REGEXES):
+            else:
                 if file_path not in self.ignored_files:
```

Comment:
let's add an `else` to this. I think we should pop an error that the file did no match any pattern..